### DISCUSSION TITLE
No IFR clearance with a FP discontinuity?

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#6227] [Help]
If I init with a Simbrief FP, then I have to clear the flightplan discontinuity's before I get the "Request IFR clearance"-option in the ATC menu.

Is this correct? Normally I would get my clearance information to finalize my FP (need runway to determine SID etc.)

**Comment by CptBayer** [comment]
normaly you talk to a controller :-p .... for him it doesn´t matter if you have discos in your flightplan..... In real life it is possible to get the clearance via datalink and ACARS... it is becoming more and more automated ... i could imagine that MS have a problem with the disco?! .... I will try this also ...
